apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/ConfigServerEurekaServerConfigure.java

```diff
@@ -29,4 +36,43 @@
 @EnableEurekaServer
 @ConditionalOnProperty(name = "apollo.eureka.server.enabled", havingValue = "true", matchIfMissing = true)
 public class ConfigServerEurekaServerConfigure {
+
+  @Order(99)
+  @Configuration
+  static class EurekaServerSecurityConfigurer extends WebSecurityConfigurerAdapter {
```

Comment:
When I open the IDEA, found that `WebSecurityConfigurerAdapter` is Deprecated

```java
 * @see EnableWebSecurity
 * @deprecated Use a {@link org.springframework.security.web.SecurityFilterChain} Bean to
 * configure {@link HttpSecurity} or a {@link WebSecurityCustomizer} Bean to configure
 * {@link WebSecurity}
```
Should use new way to configure the eureka's security?

Reply:
There are many places using `WebSecurityConfigurerAdapter` in apollo, I think we could create a new pr to change them together in the future.